src/Fantomas/CodePrinter.fs

```diff
@@ -1637,6 +1640,17 @@ and genExpr astContext synExpr ctx =
         | Paren (_, ILEmbedded r, _) ->
             // Just write out original code inside (# ... #)
             fun ctx -> !- (defaultArg (lookup r ctx) "") ctx
+
+        | Paren (lpr, SynExpr.Ident ident, rpr) when astContext.IsInsideIfThenElseExpression ->
+            let e = SynExpr.Ident ident
+
+            match ident.idText with
+            | "op_Multiply" ->
```

Comment:
This is not really the safest check. See [example](https://fsprojects.github.io/fantomas-tools/#/ast?data=N4KABGBEAmCmBmBLAdrAzpAXFSAacUiaAYmolmPAIYA2as%2BEkAxgPZwWQ2wAuYrABwD6AWQCuNHogE0AnmAC8YACwAmSCAC%2BQA).